applications/CompressiblePotentialFlowApplication/custom_elements/adjoint_base_potential_flow_element.h

```diff
@@ -140,30 +134,25 @@ class AdjointPotentialFlowElement : public Element
 
     void PrintData(std::ostream& rOStream) const override;
 
-   Element::Pointer pGetPrimalElement();
-
+    Element::Pointer pGetPrimalElement();
 
 protected:
 
-
     Element::Pointer mpPrimalElement;
 
-    void GetWakeDistances(array_1d<double,NumNodes>& distances);
-
     void GetValuesOnSplitElement(Vector& split_element_values, const array_1d<double,NumNodes>& distances);
 
+    double GetPerturbationSize();
```

Comment:
I would actually implement this function as a private function only in the AdjointFiniteDifferencePotentialFlowElement since it is only used there.

Reply:
Absolutely!